### PR TITLE
ci: attempt to fix broken ci build

### PR DESCRIPTION
**Pull request by paulstey**

### CODE REVIEW

**Review by eldu — COMMENTED** [review]
I noticed this is on node 14. Can we bump that up to like 16 or 18?
Also I think you can also upgrade to `actions/checkout@v3` and `actions/setup-node@v3` and `actions/cache@v3`

Also why recursive?

File: .github/workflows/firebase-qa-pull-request.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
           ls -la
       - name: Build
         run: |
+          export NODE_OPTIONS="--max_old_space_size=4096"
```
Comment (broarr) [inline .github/workflows/firebase-qa-pull-request.yml:33]: Huh. I didn't know about that option. Looks like "old space" is for garbage collected heap storage.

https://stackoverflow.com/questions/48387040/how-do-i-determine-the-correct-max-old-space-size-for-node-js/48392705#48392705

4GiB seems like a lot... Is this for the preview links?

File: nuxt.config.js [hunk 2]
```diff
@@ -72,6 +72,9 @@ export default {
     '@nuxtjs/style-resources',
     // https://google-analytics.nuxtjs.org
     '@nuxtjs/google-analytics',
+    
+    '@nuxt/postcss8', 
+
```
Comment (broarr) [inline nuxt.config.js:77]: Do you need the extra spaces here?

**Review by broarr — APPROVED** [review]
Two little comments, but nothing that should block merging!

File: nuxt.config.js (hunk 2 above)
Comment (paulstey) [reply nuxt.config.js:77]: Nope, I'll fix this! Thanks, @broarr!!

**Pull request merged**